assets/sites/default/settings.acquia.php

```diff
@@ -60,10 +60,15 @@
       'node/%node/moderation',
       'file/ajax',
     );
+
+    $endpoint_paths = array(
+      'api/action/datastore/search.json'
+    );
+
     // Standarize node edit paths for validation.
     $current_path = preg_replace("/\d+/", '%node', $_GET['q']);
     foreach ($admin_paths as $admin_path) {
-      if (strpos($current_path, $admin_path) === 0) {
+      if ((strpos($current_path, $admin_path) === 0) || (strpos($current_path, $endpoint_path) === 0)) {
```

Comment:
@jacintocapote we we clean this up by changing name of variable we want update memory path for to some thing like $high_memory_paths and just use one array ?